jquery.lazyload.js

```diff
@@ -126,6 +126,9 @@
                                 var elements_left = elements.length;
                                 settings.load.call(self, elements_left, settings);
                             }
+                        }).bind("error", function() {
+                            var fallback = $self.attr("data-fallback");
+                            $self.attr("src", fallback);
```

Comment:
Good idea. We could just use the original placeholder (grey box) that is displaying currently, if there is no `data-fallback`.